Source/CarthageKitTests/FrameworkExtensionsSpec.swift

```diff
@@ -0,0 +1,25 @@
+import Quick
+import Nimble
+
+class FrameworkExtensionsSpec: QuickSpec {
+	override func spec() {
+		describe("NSURL Extensions") {
+			it("should figure out if a is a subdirectory of b") {
+				guard let subject = NSURL(string: "file:///foo/bar") else { return }
+
+				guard let unrelatedScheme = NSURL(string: "http:///foo/bar/baz") else { return }
+				guard let parentDir = NSURL(string: "file:///foo") else { return }
+				guard let immediateSub = NSURL(string: "file:///foo/bar/baz") else { return }
+				guard let distantSub = NSURL(string: "file:///foo/bar/baz/qux") else { return }
+				guard let unrelatedDirectory = NSURL(string: "file:///bar/bar/baz") else { return }
```

Comment:
These shouldn't just `return`, otherwise this is a false positive, since the test isn't running at all!